frontend/src/employee-frontend/components/child-information/assistance-need/decision/AssistanceNeededDecisionForm.tsx

```diff
@@ -480,7 +513,26 @@ export default React.memo(function AssistanceNeedDecisionForm({
             selectedTitle={formState.preparedBy1?.title ?? ''}
             onChange={(employee, title) =>
               setFieldVal({
-                preparedBy1: { employeeId: employee?.id ?? null, title }
+                preparedBy1: {
+                  ...(formState.preparedBy1 ?? {
+                    phoneNumber: null
+                  }),
```

Comment:
the employee fields in `formState` should always be defined and have `phoneNumber: null` if they are fetched from the backend due to how the data structure is defined (with `@Nested`). Would `...formState.preparedBy1,` be enough?